### PR TITLE
 #906 translate titles in enterprise_mailer

### PR DESCRIPTION
**Pull request by blancnic**
translate subject of enterprise confirmation email  	 **f3fb158**
translate welcome title in enterprise_mailer 	**e24c4a8**

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -32,6 +32,11 @@ en:
         not_confirmed: Your email address could not be confirmed. Perhaps you have already completed this step?
         confirmation_sent: "Confirmation email sent!"
         confirmation_not_sent: "Could not send a confirmation email."
+  enterprise_mailer:
+    confirmation_instructions:
+      subject: "Please confirm the email address for %{enterprise}"
+    welcome:
+      subject: "%{enterprise} is now on "
```
Comment (mkllnk) [inline config/locales/en.yml]: Can you include the sitename variable as well?

      subject: "%{enterprise} is now on %{sitename}"

And then pass it in the mailer:

    subject = t('enterprise_mailer.welcome.subject',
                enterprise: @enterprise.name,
                sitename: Spree::Config[:site_name])

**Comment by mkllnk** [comment]
Looks good to me. Can you remove the trailing whitespace [Codeclimate detected](https://codeclimate.com/github/openfoodfoundation/openfoodnetwork/pull/937)?

And in the end it would be great if you could squash the commits as well.

**Comment by blancnic** [comment]
@mkllnk : I suppose I don't need to squash commits because I included your changes in my last commit.
Tell me if it's ok or not. Thank you.

**Comment by mkllnk** [comment]
I created a separate branch to merge this in https://github.com/openfoodfoundation/openfoodnetwork/commit/3e231da472d82a6353f211b5af6c38be881649d0. Thanks for that.

**Pull request closed without merging**